pkg/namespaces/namespaces.go

```diff
@@ -380,7 +382,31 @@ func (n NetworkMode) IsBridge() bool {
 
 // IsSlirp4netns indicates if we are running a rootless network stack
 func (n NetworkMode) IsSlirp4netns() bool {
-	return n == slirpType
+	return n == slirpType || strings.HasPrefix(string(n), slirpType + ":")
+}
+
+// IsPortForwardViaRootlessKit indicates if we are doing rootless port-forwarding via rootlesskit/rootlessport
+func (n NetworkMode) IsPortForwardViaRootlessKit() bool {
+        if !n.IsSlirp4netns() {
+		return false
+	} else { // below here, implied IsSlirp4netns() == true
+		if !strings.Contains(string(n), ":") {
+			return true // default type
+		} else { // below here, also contains(":") == true
+			parts := strings.SplitN(string(n), ":", 2)
+			return parts[1] == rlkFwdType
```

Comment:
Need to verify that len(parts) == 2 else this could segfault

Reply:
just for my clarification, should i be checking if it's _exactly_ length 2? or is the fact that this statement falls on the other side of the if statement _else_ clause, where it definitely has to contain a ":" not sufficient?